test/functional/feature_dst20.py

```diff
@@ -787,27 +787,48 @@ def run_test(self):
         )
 
         # Contract ABI
-        # Temp. workaround
-        self.abi = open(
-            f"{os.path.dirname(__file__)}/../../lib/ain-contracts/dst20/output/abi.json",
-            "r",
-            encoding="utf8",
-        ).read()
-        self.reserved_bytecode = json.loads(
-            open(
-                f"{os.path.dirname(__file__)}/../../lib/ain-contracts/system_reserved/output/bytecode.json",
+        if os.getenv("BUILD_DIR"):
+            build_dir = os.getenv("BUILD_DIR")
+            self.abi = open(
+                f"{build_dir}/ain_contracts/dst20/abi.json",
                 "r",
                 encoding="utf8",
             ).read()
-        )["object"]
-
-        self.bytecode = json.loads(
-            open(
-                f"{os.path.dirname(__file__)}/../../lib/ain-contracts/dst20/output/bytecode.json",
+            self.bytecode = json.loads(
+                open(
```

Comment:
Let's abstract the path finder into a function, instead of if else of the whole codepath.